assets/js/modules/tagmanager/components/common/WebContainerSelect.js

```diff
@@ -88,11 +83,7 @@ export default function WebContainerSelect( { hasModuleAccess } ) {
 		[ containerID, setContainerID, setInternalContainerID, viewContext ]
 	);
 
-	if ( isPrimaryAMP ) {
-		return null;
-	}
-
-	const label = isSecondaryAMP
+	const label = ampMode
```

Comment:
Looks like there's no scenario where it would render as only container, as we're rendering Web Container unconditionally. I'll remove this logic so it is always `AMP Container`.